src/components/media-types/vector/index.js

```diff
@@ -34,7 +34,8 @@ export const VectorComponent = ({ src, interactive, preview, token_info }) => {
       <iframe
         title="hic et nunc SVG renderer"
         src={iframeSrc}
-        sandbox="allow-scripts"
+        sandbox="allow-scripts allow-same-origin"
```

Comment:
We CAN NOT have allow-scripts and allow-same-origin since that's basically the same as disabling the sandbox

Reply:
It would allow to access the injected wallet, possibly hacking or click-jacking and stealing funds

Reply:
missed that nuance. I'll do some testing tomorrow to see if the HTML version works with camera now rather than SVG as is. Pushed to remove allow-same-origin.